sound/soc/codecs/rt700.c

```diff
@@ -1210,6 +1215,11 @@ int rt700_io_init(struct device *dev, struct sdw_slave *slave)
 	if (rt700->hs_jack)
 		rt700_jack_init(rt700);
 
+	if (rt700->first_init)
+		regcache_cache_bypass(rt700->regmap, false);
```

Comment:
same comment as above, I am not sure about the flow and interaction with resume and loss of sync cases.